src/fluree/db/ledger/transact/error.clj

```diff
@@ -80,9 +82,11 @@
           hash-flake       (tx-meta/generate-hash-flake flakes tx-state)
           all-flakes       (conj flakes hash-flake)
           fast-forward-db? (:tt-id db-before)
-          db-after         (if fast-forward-db?
-                             (<? (dbproto/-forward-time-travel db-before all-flakes))
-                             (<? (dbproto/-with-t db-before all-flakes)))
+          db-after         (->> (if fast-forward-db?
+                                  (<? (dbproto/-forward-time-travel db-before all-flakes))
```

Comment:
Fix is to use db-root from the tx-state here. Then, you don't need to call dbproto/-rootdb. Also, there should be no reason to use tx-util/make-candidate-db - that fn is for making a proposed db before processing smartfunctions so it can clear from cache if a failure occurs. In this case, we are done processing and can proceed as we know the new db is final.

Reply:
This was resolved with Brian's check-in.